Dockerfile

```diff
@@ -1,8 +1,24 @@
 # syntax=docker/dockerfile:1
 
+FROM ghcr.io/linuxserver/baseimage-alpine:3.17 as buildstage
+
+ARG MOD_VERSION=master
+
+RUN \
+  echo "**** grab rffmpeg ****" && \
+  mkdir -p /root-layer/usr/local/bin/ && \
+  curl -o \
```

Comment:
Please make this `curl -fo` so the build fails when GitHub is not reachable or errors out